### PR TITLE
Avoid doubly expiring breakpoints

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@GoogleCloudPlatform/node-team PTAL.

File: test/standalone/test-debuglet.js [hunk 1]
```diff
@@ -311,6 +311,65 @@ describe(__filename, function(){
     debuglet.start();
   });
 
+  // This test catches regressions in a bug where the agent would
+  // re-schedule and already expired breakpoint to expire if the
```
Comment (DominicKramer) [inline test/standalone/test-debuglet.js]: You accidentally wrote 'and' instead of 'an'.

**Comment by DominicKramer** [comment]
LGTM

File: test/standalone/test-debuglet.js (hunk 1 above)
Comment (matthewloring) [inline test/standalone/test-debuglet.js]: Thanks! Good catch.

**Pull request merged**